framework/src/wifi_manager/Kconfig

```diff
@@ -11,4 +11,5 @@ config WIFI_MANAGER
 
 if WIFI_MANAGER
 source "$EXTERNALDIR/slsi_wifi/Kconfig"
+source "$EXTERNALDIR/wpa_supplicant/Kconfig"
```

Comment:
I think wps_supplicant should belong to "Wireless" in menuconfig tree as below:
in os/net/Kconfig
....
menu "Wireless"
source ../framework/src/wifi_manager/Kconfig
source "$EXTERNALDIR/wpa_supplicant/Kconfig"
endmenu #Wireless

Reply:
Let's modify this like below:
Enable   WiFi Manager |   |   |  
-- | -- | -- | --
  | slsi_wifi |   | choice
  | wpa_supplicant | select by slsi_wifi

Reply:
In commit, 5e667f8, I added dependancy.